### ISSUE TITLE
Have you tried doing it in Drupal?

### ISSUE THREAD

**Issue by noels** [SVEllipsis/wordpress#1]

**Comment by kevinprince** [comment]
yeah, but then i got a cost estimate and a suggestion to use someone gat1 ?

**Comment by noels** [comment]
I think there's a module for that.
Kevin Prince wrote on 11/05/2015 21:34:

**Comment by theothertom** [comment]
Have you checked out https://www.drupal.org/project/marine_tracking yet?
